src/io_utils.rs

```diff
@@ -83,3 +86,37 @@ pub(crate) fn read_payment_info(config: &Config) -> Vec<PaymentInfo> {
 
 	payments
 }
+
+/// Provides an interface that allows a previously persisted key to be unpersisted.
+pub trait KVStoreUnpersister {
```

Comment:
I wonder if a persister is the right abstraction. Seems we want some abstract data storage where persistence is more of an implementation detail?

Reply:
@G8XSU How does VSS deal with removed data?

Reply:
currently vss doesn't have a delete/remove data api. 
from my understanding, ldk-node uses del/remove for events, i am not sure if it will be using vss for that. 
Also in later commit i see that unpersist was removed.